src/util/Util.cpp

```diff
@@ -165,3 +166,17 @@ gboolean Util::paintBackgroundWhite(GtkWidget* widget, cairo_t* cr, void* unused
 	return false;
 }
 
+char* Util::getCoordinateString(double xVal, double yVal)
+{
+	char tmpX[G_ASCII_DTOSTR_BUF_SIZE];
+	g_ascii_formatd(tmpX, G_ASCII_DTOSTR_BUF_SIZE, Util::getFormatdPrecisionString(), xVal);
+	char tmpY[G_ASCII_DTOSTR_BUF_SIZE];
+	g_ascii_formatd(tmpY, G_ASCII_DTOSTR_BUF_SIZE, Util::getFormatdPrecisionString(), yVal);
+
+	return g_strdup_printf("%s %s", tmpX, tmpY);
+}
+
+char* Util::getFormatdPrecisionString()
+{
+	return g_strdup_printf("%%.%df", PRECISION_DECIMAL_PLACES);
```

Comment:
Avoid rebuilding this format string every time.
The only use of  PRECISON_DECIMAL_PLACES seems to be to build this string.  So it makes sense to define a char* constant instead.

Declare it in the header but you have to initialize it here:
`const char* Util::PRECISION_FORMAT_STRING = "%.8f";   //use 8 digits of precision   https://m.xkcd.com/2170/ `

Reply:
The xkcd link in the comment is also a nice easter egg :smile: